### ISSUE TITLE
Include directive dislikes directories

### ISSUE THREAD

**Issue by TheCodedOne** [dbaumgarten/yodk#104]
**Describe the bug**
When using include and directories, if you include a file that includes another, it attempts to include the sub-include based on the directory of the original script rather than of the script actually trying to include.

**To Reproduce**
Create 3 scripts
```
/a.nolol
/b.nolol
/c/c.nolol
```
have `b` include `a`, and `c` include `b`.
see `c` error that it cannot find `a`

**Code**

`/a.nolol`
```
hello=world
```
`/b.nolol`
```
include "a"
```
`/c/c.nolol`
```
include "../b"
```

**Expected behaviour**
Nolol compilation should include based on the local directory of the file including, or have a way of defining a root directory to base all includes from

**Platform:**
 - OS: windows
 - YODK: 0.1.9

**Comment by dbaumgarten** [comment]
You are right. Includes are currently always relative to the main-file, which causes issues with nested includes accross multiple directories.
Making all includes relative to the current file would probably be the best solution.

**Comment by dbaumgarten** [comment]
Published as part of v0.1.10
